addins/upgrade/upgrade_shared.html

```diff
@@ -26,6 +26,21 @@
 			function getSharedValueIntoUI() {
 				document.getElementById("getValue").innerText = g_sharedState.value;
 			}
+
+			function setStartupStateValueFromUI() {
+				const startupState = document.getElementById("setStartupStateValue").value;
+				OfficeRuntime.currentRuntime.setStartupState(startupState);
+			}
+
+			function getStartupStateValueIntoUI() {
+				const startupState = OfficeRuntime.currentRuntime.getStartupState();
+				document.getElementById("getCurrentStateValue").innerText = startupState;
+			}
+
+			function getCurrentStatValueIntoUI() {
+				const currentState = OfficeRuntime.currentRuntime.getState();
+				document.getElementById("getCurrentStateValue").innerText = currentState;
+			}
```

Comment:
Please make your spacing visible. I suspect something is changing that this web differ is not showing me.

Reply:
I double checked the spacing. I think they should be correct now.